### PR TITLE
Add missing propertymapping for eav attribute creation/updates

### PR DESCRIPTION
**Pull request by brosenberger**
The method \Magento\Eav\Setup\EavSetup::addAttribute parses the input data for an attribute added/updated with a update script / patch and removes all not configured attributes - therefore all attributes columns that are added by this module can not be updated programmatically.

Within this pull request I added the created columns with their respective default values

### CODE REVIEW

File: src/module-elasticsuite-catalog/Model/ResourceModel/Setup/PropertyMapper.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Economix\Elasticsuite\Model\ResourceModel\Setup;
```
Comment (romainruaud) [inline src/module-elasticsuite-catalog/Model/ResourceModel/Setup/PropertyMapper.php:3]: Namespace is incorrect.

**Comment by romainruaud** [comment]
Hi @brosenberger 

This seems interesting but the namespace of the class is incorrect.

And the integration tests are not passing due to the following error : 

```
Property mapper Economix\Elasticsuite\Model\ResourceModel\Setup\PropertyMapper must implement \Magento\Eav\Model\Entity\Setup\PropertyMapperInterface
```

**Comment by brosenberger** [comment]
@romainruaud i fixed the namespace issue, but integration tests are not starting (php code quality check run through) - is anything else needed for that?

File: src/module-elasticsuite-catalog/Model/ResourceModel/Setup/PropertyMapper.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+     * {@inheritdoc}
+     */
+    public function map(array $input, $entityTypeId)
+    {
+        return [
+            'is_displayed_in_autocomplete' => $this->_getValue($input, 'is_displayed_in_autocomplete', 0),
+            'is_used_in_spellcheck' => $this->_getValue($input, 'is_used_in_spellcheck', 0),
+            'facet_min_coverage_rate' => $this->_getValue($input, 'facet_min_coverage_rate', 90),
+            'facet_max_size' => $this->_getValue($input, 'facet_max_size', 10),
+            'facet_sort_order' => $this->_getValue($input, 'facet_sort_order', 25),
```
Comment (romainruaud) [inline src/module-elasticsuite-catalog/Model/ResourceModel/Setup/PropertyMapper.php:37]: should be `Smile\ElasticsuiteCore\Search\Request\BucketInterface::SORT_ORDER_COUNT` not `25`

File: src/module-elasticsuite-catalog/Model/ResourceModel/Setup/PropertyMapper.php [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            'display_pattern' => $this->_getValue($input, 'display_pattern', 10),
+            'display_precision' => $this->_getValue($input, 'display_precision', 0),
+            'sort_order_asc_missing' => $this->_getValue($input, 'sort_order_asc_missing', 10),
```
Comment (romainruaud) [inline src/module-elasticsuite-catalog/Model/ResourceModel/Setup/PropertyMapper.php:40]: should be `_last` not `10`

File: src/module-elasticsuite-catalog/Model/ResourceModel/Setup/PropertyMapper.php [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+            'sort_order_desc_missing' => $this->_getValue($input, 'sort_order_desc_missing', 10),
```
Comment (romainruaud) [inline src/module-elasticsuite-catalog/Model/ResourceModel/Setup/PropertyMapper.php:41]: same here, should be `_last` not `10`

File: src/module-elasticsuite-catalog/Model/ResourceModel/Setup/PropertyMapper.php [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+            'facet_boolean_logic' => $this->_getValue(
+                $input,
+                'facet_boolean_logic',
+                \Smile\ElasticsuiteCore\Api\Index\Mapping\FieldInterface::FILTER_LOGICAL_OPERATOR_OR
+            ),
+            'is_display_rel_nofollow' => $this->_getValue($input, 'is_display_rel_nofollow', '0'),
```
Comment (romainruaud) [inline src/module-elasticsuite-catalog/Model/ResourceModel/Setup/PropertyMapper.php:47]: `0` not `'0'`

**Comment by romainruaud** [comment]
@brosenberger I've reviewed this last week, I'd like to be able to integrate it in both 2.10.x and master branch for the next release (around march 14th).

Do you think you'll be able to proceed with the asked fixes or we should go through them by our own ?

Regards

**Pull request merged**